### ISSUE TITLE
[Adaptive Card-Schema Explorer-Media]: Visual name and accessible name is different for more options button.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7915]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)

Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3.Navigate to "Media" link present under Card elements from the left navigation using the tab key and activate it using Enter key.
4. Navigate to the adaptive cards using the Tab key.
5. Navigate to the Adaptive card video content present under Adaptive Card using Tab key.
6. Activate it using Enter key.
7. Observe the issue on "More Options" button.

Actual Result:
Visual name and accessible name are different for more options button as "Show more media controls".

Expected Result:
Visual name and accessible name have to be same for more options button.

Observation:
It impacts the voice access users as not able to access the more options button.

User Experience:
Keyboard users and cognitive users and Voice Access users will be impacted and feel stressed to access the button if Visual name and accessible names are different for more options button as "Show more media controls".

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.05.03#

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="929" alt="Visual name and accessible name issue on more option button" src="https://user-images.githubusercontent.com/104482612/191680782-8a86132c-a69a-491c-90e9-b5e171cbf804.png">

https://user-images.githubusercontent.com/104482612/191682678-acba1241-64a4-48fc-8914-40441691cec1.mp4

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hey @Susmitha474,

I am closing this issue because I believe it is an bug with Edge.

Our `Media` element utilizes [HTML's `<video/>`](https://developer.mozilla.org/en-US/docs/web/html/element/video).

I inspected the "More options" button on the Mozilla website, and the button also has a different visual name and accessible name (see picture below).

![Screenshot (3)](https://user-images.githubusercontent.com/98650930/198748609-b67e77ed-83b5-4e3f-8e13-848df8cf1ff5.png)

Since we don't implement the control ourselves, we cannot update the names to match.

I'll follow up with Edge so that we can get this resolved. Let me know if you have any questions - thanks!
